packages/@orchidui-vue/src/Overlay/CopyTooltip/OcCopyTooltip.vue

```diff
@@ -1,17 +1,25 @@
 <script setup lang="ts">
 import { Icon, Tooltip } from "@/orchidui";
 
-defineProps({
-  value: String,
+const props = defineProps({
+  value: [String, Blob],
   tooltipText: {
     type: String,
     default: "Copied!",
   },
   tooltipOptions: Object,
 });
-const copyToClipboard = async (text) => {
+const copyToClipboard = async (data) => {
   try {
-    await navigator.clipboard.writeText(text);
+    if (props.value instanceof Blob) {
```

Comment:
You could delete `props.value` and use `data` instead and delete `const props` above